config/hint.yml

```diff
@@ -13,8 +13,13 @@ hint:
     config: "hint_config"
   expose: true
 
+hintr-loadbalancer:
+  tag: "main"
+  api_instances: 1
```

Comment:
Do you want to increase this a bit since we feel this works ok now? Possibly varies by machine?